Frameworks/SafariServices/SFSafariWebViewController.mm

```diff
@@ -177,4 +207,8 @@ - (void)_cancel {
     [self dismissViewControllerAnimated:NO completion:nil];
 }
 
+- (void)dealloc {
+    _webView.delegate = nil;
```

Comment:
True, but the UIWebView docs specify very sternly that the delegate field must be reset to nil before deallocating the UIWebView, so maybe I should just comply?

In looking at this question, I did discover that UIWebView has a retain cycle involving its callbacks, so that's something that needs to be fixed.

---
In reply to: [83337136](https://github.com/Microsoft/WinObjC/pull/1147#discussion_r83337136) [](ancestors = 83337136)